src/routes/userRoute.js

```diff
@@ -0,0 +1,12 @@
+import express from 'express';
+import UserController from '../controllers/userController';
+import verify from '../helpers/verifyToken';
+import upload from '../helpers/profilePic';
+
+const router = express.Router();
+
+router.get('/auth/login', UserController.login);
+router.post('/auth/logout', UserController.logOut);
+router.put('/user', verify, upload.single('avatar'), UserController.updateProfile);
```

Comment:
A middleware should be created to validate the users request body and cover edge cases. For example, the username is meant to be unique, and if not validated it would break the application.